stationsim_model/source/Model.cpp

```diff
@@ -87,18 +87,18 @@ namespace station_sim {
     }
 
     void Model::set_boundaries() {
-        boundaries[0].x = 0;                                   // x1
-        boundaries[0].y = 0;                                   // y1
-        boundaries[1].x = model_parameters.get_space_width();  // x2
-        boundaries[1].y = model_parameters.get_space_height(); // y2
+        boundaries = model_parameters.get_boundaries();
```

Comment:
`boundaries` are now the vertices of an arbitrary 2D polygon, right?

Reply:
That's correct.  If you have a better name in mind, do share!

Reply:
`boundary_vertices`, maybe?